imports/plugins/core/cart/server/no-meteor/resolvers/Query/getCommonOrderForCartGroup.js

```diff
@@ -0,0 +1,22 @@
+import { decodeCartOpaqueId, decodeFulfillmentGroupOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/cart";
+
+/**
+ * @name Query.getCommonOrderForCartGroup
+ * @method
+ * @memberof Cart/GraphQL
+ * @summary Get a CommonOrder object for a cart fulfillment group
+ * @param {Object} parentResult - unused
+ * @param {ConnectionArgs} args - An object of all arguments that were sent by the client
+ * @param {String} args.cartId - cart ID to create CommonOrder from
+ * @param {String} args.fulfillmentGroupId - fulfillment group ID to create CommonOrder from
+ * @param {Object} context - An object containing the per-request state
+ * @returns {Promise<Object>|undefined} A CommonOrder object
+ */
+export default async function getCommonOrderForCartGroup(parentResult, args, context) {
+  const { cartId, fulfillmentGroupId } = args;
+
+  return context.queries.getCommonOrderForCartGroup(context, {
+    cartId: decodeCartOpaqueId(cartId),
+    fulfillmentGroupId: decodeFulfillmentGroupOpaqueId(fulfillmentGroupId)
+  });
+}
```

Comment:
Sorry for not being clear. I don't think it's a good idea to have this query in GraphQL. It should be for internal use only. Maybe this would change at some point but for now CommonOrder is more of an internal concept among plugins that doesn't need to be part of the GraphQL schema.